plugins/TagFix_BadValue.py

```diff
@@ -111,11 +111,10 @@ def check(self, data, tags, check_list_open):
         keys = set(keyss) & check_list_open
         for k in keys:
             if not self.Values_open.match(tags[k]):
-                if k in self.exceptions_open:
-                    if tags[k] in self.exceptions_open[k]:
-                        # no error if in exception list
-                        continue
-                err.append({"class": 3040, "subclass": stablehash64(k), "text": T_("Concerns tag: `{0}`", '='.join([k, tags[k]])) })
+                if (not k in self.exceptions_open or # no whitelists exist for the key
+                   (not ";" in tags[k] and not tags[k] in self.exceptions_open[k]) or # single-valued tag and not whitelisted
+                   (";" in tags[k] and any(map(lambda val: val not in self.exceptions_open[k] and not self.Values_open.match(val), tags[k].split(";"))))): # multi-valued tag and one or more bad values
+                    err.append({"class": 3040, "subclass": stablehash64(k), "text": T_("Concerns tag: `{0}`", '='.join([k, tags[k]])) })
```

Comment:
My reasoning against that was that it's probably much more efficient to use a `a in b` (as I guess the majority of the cases is without `;`) rather than the full call with map, split, match, but I can change it, no problem!

Reply:
Done, also the requested combination with the previous 'if'. Fortunately there's no limit on the number of parentheses :)